.ci/atime/tests.R

```diff
@@ -119,6 +119,22 @@ test.list <- atime::atime_test_list(
       data.table:::setDT(L)
     },
     Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12", # Parent of the first commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801)
-    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15") # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15"),# Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+
+  #issue reported in: https://github.com/Rdatatable/data.table/issues/5424
+  #Fixed in: https://github.com/Rdatatable/data.table/pull/5463
```

Comment:
why does line 122 say PR #5427 fixes the issue, but line 125 says this is fixed in #5463? Please double check / fix so that these comments are consistent.

Reply:
@tdhock the line 122 is different test which was added already done  by @Anirban166 and has been merged, I added test from line 124 and named ".N and := regression fixed in #5463", for review so that it can also be merged together with the first two

Reply:
ok